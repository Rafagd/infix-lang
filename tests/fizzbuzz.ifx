index is i32;
index = 0;

(index < argc) repeat {
    {
        ([[argv @ index] @ 0] == ["f" @ 0]) ? {
            void print "Fizz";
        }
    } ? {
        void print "Buzz";
    };
    index = index + 1;
};
