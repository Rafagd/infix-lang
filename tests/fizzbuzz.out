Buzz
Fizz
Fizz
Buzz
Fizz
